### PR TITLE
Added creation column

### PR DESCRIPTION
**Pull request by duncanp-sonar**
Related to #2042

TODO: sorting

### CODE REVIEW

File: src/IssueViz.Security/Taint/TaintList/TaintIssuesControl.xaml [hunk 1]
```diff
@@ -117,6 +117,14 @@
                         </DataTemplate>
                     </DataGridTemplateColumn.CellTemplate>
                 </DataGridTemplateColumn>
+                <DataGridTemplateColumn CanUserSort="True" SortMemberPath="TaintIssueViz.Issue.CreationTimestamp" Header="Created" Width="138"  MinWidth="100">
```
Comment (rita-g-sonarsource) [inline src/IssueViz.Security/Taint/TaintList/TaintIssuesControl.xaml:120]: Minor: "Created at" ? "Creation time"? Can be "created" as well, I can't think of a popular example to follow.

File: src/IssueViz.Security/Taint/TaintIssueToIssueVisualizationConverter.cs [hunk 2]
```diff
@@ -76,6 +76,8 @@ private IAnalysisIssueBase ConvertToAnalysisIssue(SonarQubeIssue sonarQubeIssue)
                 sonarQubeIssue.TextRange.EndOffset,
                 sonarQubeIssue.Hash,
                 Convert(sonarQubeIssue.Severity),
+                sonarQubeIssue.CreationTimestamp,
```
Comment (rita-g-sonarsource) [inline src/IssueViz.Security/Taint/TaintIssueToIssueVisualizationConverter.cs:79]: Update the converter tests too :)

File: src/IssueViz.Security/Taint/TaintList/TaintIssuesControl.xaml [hunk 3]
```diff
@@ -117,6 +117,14 @@
...
+                    <DataGridTemplateColumn.CellTemplate>
+                        <DataTemplate>
+                            <TextBlock Text="{Binding TaintIssueViz.Issue.CreationTimestamp, StringFormat=G}"
```
Comment (duncanp-sonar) [inline src/IssueViz.Security/Taint/TaintList/TaintIssuesControl.xaml:123]: Display in general local date-and-time format. I considered just showing the date, but decided on date and time. We can tweak the format if necessary.

**Review by rita-g-sonarsource — APPROVED** [review]

File: src/IssueViz.Security/Taint/TaintList/TaintIssuesControl.xaml (hunk 3 above)
Comment (rita-g-sonarsource) [reply src/IssueViz.Security/Taint/TaintList/TaintIssuesControl.xaml:123]: I think the most polished would be the user-friendly texts, e.g. "6 months ago". I don't know if there's any built-in way to do that. We can tweak it later if needed.

**Pull request merged**